### ISSUE TITLE
Vue icons not showing in latest v1.3.0

### ISSUE THREAD

**Issue by robertmoura** [Monokai/monokai-pro-vscode#404]
Updating to the latest Monokai Pro version (v1.3.0) in VSCode seems to have changed the Vue icon to a generic one.

I use the following settings:
```json
{
  "workbench.colorTheme": "Monokai Pro (Filter Spectrum)",
  "workbench.iconTheme": "Monokai Pro (Filter Spectrum) Icons",
}
```

Here's an image of the issue:

<img width="222" alt="Screenshot 2024-04-18 at 3 54 49 PM" src="https://github.com/Monokai/monokai-pro-vscode/assets/17804984/8f26394d-e1c3-4046-b6a0-07c31f13fa96">

Really liking the new icons by the way 😊

**Comment by QInJ1995** [comment]
Same here

**Comment by alexmanzo-fundrise** [comment]
+1, same issue for me as well

**Comment by Monokai** [comment]
Fixed in 1.3.1

**Comment by Torrent007** [comment]
Thanks dude, for quick fix and all the works ;)

**Comment by robertmoura** [comment]
Thanks soo much! This is fixed in v1.3.1 😊
